lib/records.js

```diff
@@ -191,11 +203,12 @@ class RecordsDecoder extends Transform {
    *
    * @param {object} options - The initialization options.
    * @param {string} options.compression - The kind of compression to use in records data.
+   * @param {string} options.shouldParseJson - If data is in JSON format and should be parsed.
    * @param {string} options.logger - An instance of a logger.
    * @param {object} options.s3Client - The s3Client in the current kinesis client.
    * @param {boolean} options.useS3ForLargeItems - Whether to automatically use an S3 bucket to store large items or not.
    */
-  constructor({ compression, logger, s3Client, useS3ForLargeItems }) {
+  constructor({ compression, logger, s3Client, shouldParseJson = 'auto', useS3ForLargeItems }) {
```

Comment:
This is already defaulted somewhere else.

Reply:
fixed